changelog.md

```diff
@@ -0,0 +1,19 @@
+
+# Change Log
+All notable changes to this project will be documented in this file.
+ 
+The format is based on [Keep a Changelog](http://keepachangelog.com/)
+and this project adheres to [Semantic Versioning](http://semver.org/).
+ 
+## [0.1.1] - 2023-03-15
```

Comment:
```suggestion
## Unreleased
``` 

We should keep the next version always as `Unreleased` because we don't know if it will be a patch or a minor release. This will always depend on which commits are merged into the `main` branch (if they contain a breaking change or only bug fixes, for example).

Then, when preparing the next release, we will look at the diff from the previous tag to the `main` branch `HEAD` and decide what's the next semver to use, updating the Changelog file accordingly.

I'll add an issue to create a document with the release strategy for this repo so we have this process saved.